pkg/operator/ceph/cluster/osd/labels.go

```diff
@@ -52,6 +52,7 @@ func (c *Cluster) getOSDLabels(osd OSDInfo, failureDomainValue string, portable
 	labels[OsdIdLabelKey] = stringID
 	labels[FailureDomainKey] = failureDomainValue
 	labels[portableKey] = strconv.FormatBool(portable)
+	labels["device-type"] = osd.DeviceType
```

Comment:
```suggestion
	labels["device-type"] = osd.DeviceType
```
can we not use hard-coded and declare this somewhere similar to other labels above?

Reply:
okay